src/aframe-streetmix-parsers.js

```diff
@@ -692,16 +694,15 @@ function createSegmentElement (scaleX, positionY, rotationY, mixinId, length, re
 
   const scalePlane = scaleX + ' ' + scaleY + ' 1';
   const scaleBox = scaleX + ' 1 1';
+  let heightLevels = [0.2, 0.4, 0.8]
```

Comment:
confused why there are 3 heigh levels, aren't there only 2 potential values for elevation 0 and 1? @Algorush